README.md

```diff
@@ -195,6 +196,35 @@ To a new image and apply it to the CheCluster, run:
 yarn build-and-patch
 ```
 
+### Update dashboard on remote cluster using `skaffold.dev`
+
+To update the dashboard deployment you need access to the Kubernetes cluster (see [Running locally against remote Che Cluster (Node.js v.16)](#running-locally-against-remote-che-cluster-nodejs-v16))
+
+Then proceed with the following steps: 
+
+```sh
+# export an environment variable to define a repository you want images to be pushed, e.g.:
+export DEFAULT_REPO=quay.io/<project-id>
```

Comment:
I suggest using existing environment variables:
https://github.com/eclipse-che/che-dashboard/blob/main/run/build-and-patch.sh#L16
```
DEFAULT_REPO="${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}"
```
WDYT?

Reply:
done